src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,26 @@
-export const TodoInfo = () => {};
+import { getUserById } from '../../services/userService';
```

Comment:
According to the checklist, you should prepare data in one place (App component) and pass it to child components. Avoid using the `import` keyword in non-root components. Consider fetching the user data in the App component and passing it as a prop to `TodoInfo`.